src/test_l1_tvSettings.c

```diff
@@ -345,7 +311,7 @@ void test_l1_tvSettings_positive_RegisterVideoFormatChangeCB (void)
 	callbackData.cb = tvVideoFormatChangeHandler;
 
 	result = RegisterVideoFormatChangeCB(&callbackData);
-	UT_ASSERT_AUTO_TERM_NUMERICAL(result, tvERROR_NONE);
+	UT_ASSERT_EQUAL(result, tvERROR_NONE);
 	if(callbackData.userdata){
 		free(callbackData.userdata);
 		callbackData.userdata = NULL;
```

Comment:
line no : 309 use strncpy instead strcpy

Reply:
Updated all strcpy in file to strncpy